cfgov/v1/models/base.py

```diff
@@ -131,6 +131,14 @@ class CFGOVPage(Page):
         ObjectList(settings_panels, heading='Configuration'),
     ])
 
+    def get_authors(self):
+        """ Returns a sorted list of authors. Default is alphabetical """
+        return self.alphabetize_authors()
+
+    def alphabetize_authors(self):
+        """ Alphabetize authors of this page by last name """ 
+        return sorted(self.authors.names(), key=lambda x: x.split(" ")[-1])
```

Comment:
yeah, that's a good point.  so perhaps use a trim() before the split.

also, i don't think we're accounting for if the last names are the same.  i.e., i believe there should be a sort ordering - last name, then first name.